pkg/api/validation/validation.go

```diff
@@ -119,6 +119,16 @@ func validateGCEPersistentDisk(PD *api.GCEPersistentDisk) errs.ValidationErrorLi
 	return allErrs
 }
 
+func ValidatePersistentVolume(persistentvolume *api.PersistentVolume) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+	return allErrs
```

Comment:
So long as an administrator is able to create the devices available for storage and the kubelet is able to mount the filesystem, it should be possible to support many different types of storage devices.  The initial focus will be on cloud volumes (GCE/AWS -- which also require attaching) and NFS as an example of a non-cloud provider.